cluster/hosts.go

```diff
@@ -54,14 +56,37 @@ func (c *Cluster) TunnelHosts(ctx context.Context, flags ExternalFlags) error {
 		return err
 	}
 	for _, host := range c.InactiveHosts {
-		log.Warnf(ctx, "Removing host [%s] from node lists", host.Address)
-		c.EtcdHosts = removeFromHosts(host, c.EtcdHosts)
-		c.ControlPlaneHosts = removeFromHosts(host, c.ControlPlaneHosts)
-		c.WorkerHosts = removeFromHosts(host, c.WorkerHosts)
-		c.RancherKubernetesEngineConfig.Nodes = removeFromRKENodes(host.RKEConfigNode, c.RancherKubernetesEngineConfig.Nodes)
+		c.removeHostFromNodeList(ctx, host)
 	}
 	return ValidateHostCount(c)
+}
+
+func (c *Cluster) RemoveHostsLabeledToIgnoreUpgrade(ctx context.Context) {
+	kubeClient, err := k8s.NewClient(c.LocalKubeConfigPath, c.K8sWrapTransport)
+	nodes, err := kubeClient.CoreV1().Nodes().List(metav1.ListOptions{})
+	if err != nil {
+		// not returning error since upgrade can proceed
+		log.Infof(ctx, "[RemoveHostsLabeledToIgnoreUpgrade] Error listing nodes: %v", err)
```

Comment:
Can this be changed to inform the user what we are doing based on the fact we can't get nodes? e.g. "Error listing nodes but continuing upgrade"

Can we also keep the logging in either `[plane] message` or just `message` and keep the function log lines for debug?